Trimble.Modus.Components/Hosting/AppBuilderExtensions.cs

```diff
@@ -73,5 +74,32 @@ public static MauiAppBuilder ConfigurePopups(this MauiAppBuilder builder, Action
                 });
             return builder;
         }
+        public static void AllowMultiLineTruncation()
+        {
+            static void UpdateMaxLines(Microsoft.Maui.Handlers.LabelHandler handler, ILabel label)
+            {
+#if ANDROID
```

Comment:
Can you also check in Windows? We might (or maynot) probably need something like this for the windows too.

Reply:
Yes will check

Reply:
Is this also done?

Reply:
I tried for windows but it doesn't work as it does for iOS and android ,but tail truncate works hence it adds ellipsis to the end of the first line , if the message we pass is entered with 3 lines (eg. line1/n line2/n line 3...) it adds ellipsis
To the third line

Reply:
The Tablet's Toast view is not replicated in the desktop version as there is no specifics given for it, the ellipsis in windows is not set as expected after three lines its added to end of each line .
![Screenshot_1685950806](https://github.com/trimble-oss/modus-mobile-maui-components/assets/122255626/5a4243af-75b6-4f3c-ad17-69f7bf53eb31)
![desktop_screen](https://github.com/trimble-oss/modus-mobile-maui-components/assets/122255626/9ab17c4d-f2d3-41a2-a3e9-406c75cc05a2)
![desktopToast](https://github.com/trimble-oss/modus-mobile-maui-components/assets/122255626/f77d9ee8-70a3-454f-b932-c09955057e7b)